hl7-mllp/2.1/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,35 @@
+= HL7 MLLP Connector 2.1 - Mule 4
+:page-aliases: connectors::hl7/hl7-mllp-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector HL7 Minimal Lower Layer Protocol (MLLP) Connector (HL7 MLLP) provides connectivity and parsing functionality for HL7 MLLP messages.
+
+For compatibility information and fixed issues, see the HL7 MLLP Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+
+* Mule runtime engine (Mule)
+
+* Elements and global elements in a Mule flow
+
+* Creating a Mule app using Anypoint Studio (Studio)
+
+* HL7 MLLP Connector
+
+To use HL7 MLLP Connector, you must have:
+
+* Purchased a MuleSoft license for HL7. Contact your Account Representative for information on purchasing a license.
```

Comment:
Reformat per SR and reword for parallelism:
```suggestion
* A MuleSoft license for HL7
+
Contact your Account Representative for information about purchasing a license.
```

Reply:
fixed.